src/components/Wizard/CreateImageWizard.js

```diff
@@ -116,7 +119,8 @@ const CreateImageWizard = (props) => {
         };
       } else if (
         formValues?.image?.type === "vmdk" ||
-        formValues?.image?.type === "ova"
+        formValues?.image?.type === "ova" ||
+        formValues?.image?.type === "edge-vsphere"
```

Comment:
Same as the above here:
https://github.com/osbuild/osbuild-composer/blob/dc389eaa71936790349577b81281fae7ff4c5d9e/test/cases/ostree-vsphere.sh#L491